Lesson34/feyza/lesson34-app/src/userContext.jsx

```diff
@@ -0,0 +1,35 @@
+import { createContext, useReducer, useState } from "react";
+
+export const UserContext = createContext();
+export const UserDispatchContext = createContext();
+
+export const UserProvider = ({ children, initialState }) => {
+  const [user, dispatch] = useReducer(userReducer, initialState ?? {});
+
+  return (
+    <UserContext.Provider value={loggedOutUser}>
```

Comment:
```suggestion
    <UserContext.Provider value={user}>
```

Use 'user' which is a current user state in the application since we want to pass up to date user data in to all component in the application.